lib/streamlit/web/server/server.py

```diff
@@ -381,7 +385,6 @@ def _create_app(self) -> tornado.web.Application:
                             ],
                         },
                     ),
-                    (make_url_path_regex(base, trailing_slash=False), AddSlashHandler),
```

Comment:
This seems this was added to fix a problem with baseUrlPath a long time ago, maybe double-check that the `baseUrlPath` usage described in the issue is still possible:

https://github.com/streamlit/streamlit/pull/528